README.md

```diff
@@ -28,9 +28,26 @@ linter:
 ```
 you could lint your package like this:
 
-    $ dartanalyzer --options .analysis_options .
+    $ dartanalyzer --options analysis_options.yaml .
     
-and see any violations of the `annotate_overrides`, `hash_and_equals`, and `prefer_is_not_empty` rules in the console.  In practice you would probably configure quite a few more rules (the full list is [here][lints]). For more on configuring analysis, see the analysis option file [docs][options_file].
+and see any violations of the `annotate_overrides`, `hash_and_equals`, and `prefer_is_not_empty` rules in the console.  In practice you would probably configure quite a few more rules (the full list is [here][lints]).
+
+If a specific lint warning should be ignored, it can be flagged with a comment.  For example, 
+
+```dart
+   // ignore: avoid_as
+   (pm as Person).firstName = 'Seth'
+```
+
+tells the `dartanalyzer` to ignore this instance of `avoid_as` warning.  As lints are treated the same as errors and warnings by the analyzer, their severity can similarly be configured in an options file.  For example, an analysis options file that specifies
+
+```
+analyzer:

```

Comment:
You may also want to include the `linter:` section in this example, so users know they have to both enable the lint and configure its severity?